src/fastoad/models/performances/mission/segments/ground_speed_change.py

```diff
@@ -0,0 +1,107 @@
+"""Classes for acceleration/deceleration segments."""
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2021 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import List
+
+from scipy.constants import g
+
+from fastoad.model_base import FlightPoint
+from .base import GroundSegment
+from ..exceptions import FastFlightSegmentIncompleteFlightPoint
+
+_LOGGER = logging.getLogger(__name__)  # Logger for this module
+
+
+@dataclass
+class GroundSpeedChangeSegment(GroundSegment, mission_file_keyword="ground_speed_change"):
+    """
+    Computes a flight path segment where aircraft is accelerated or de-accelerated on the ground
+
+    The target must define an airspeed (equivalent, true or Mach) value.
+    """
+
+    def complete_flight_point(self, flight_point: FlightPoint):
+        """
+        Computes data for provided flight point.
+
+        Assumes that it is already defined for time, altitude, mass,
+        ground distance and speed (TAS, EAS, or Mach).
+
+        :param flight_point: the flight point that will be completed in-place
+        """
+        flight_point.engine_setting = self.engine_setting
+
+        self._complete_speed_values(flight_point)
+
+        flight_point.alpha = self.alpha
+
+        atm = self._get_atmosphere_point(flight_point.altitude)
+        reference_force = 0.5 * atm.density * flight_point.true_airspeed ** 2 * self.reference_area
+
+        if self.polar:
+            flight_point_copy = deepcopy(flight_point)
+            # The ground effect expects the altitude from the ground
+            flight_point_copy.altitude = flight_point_copy.altitude - self.start.altitude
+
+            modified_polar = self.polar_modifier.modify_polar(self.polar, flight_point_copy)
```

Comment:
And the second modification that would use the suggested change in [GroundEffectRaymer](https://github.com/fast-aircraft-design/FAST-OAD/pull/481/files#diff-86d2c880fe0a8ceb8ce6e2663eba2fefec24a08ab5a207d69cdd3b25deaba50e).
```suggestion
            modified_polar = self.polar_modifier.modify_polar(self.polar, flight_point)
```